src/Components/SelectedMovie/SelectedMovie.css

```diff
@@ -1,16 +1,27 @@
-
+.selectedMovie {
+  width: 80vw;
```

Comment:
I think we might run into issues with using vw as width.  I'd suggest 100% width so it matches the container's width and doesn't go out of sync, with vw it will always change based on your window size.